ios/Classes/NativeTagTechnologyProxy.m

```diff
@@ -0,0 +1,40 @@
+/**
+ * Axway Titanium
+ * Copyright (c) 2009-present by Axway Appcelerator. All Rights Reserved.
+ * Licensed under the terms of the Apache Public License
+ * Please see the LICENSE included with this distribution for details.
+ */
+
+#import "NativeTagTechnologyProxy.h"
+#import "TiBlob.h"
+#import "TiNFCTagProxy.h"
+#import "TiNfcUtilities.h"
+#import "TiUtils.h"
+
+@implementation NativeTagTechnology
+
+#pragma mark Public API's
+
+- (void)connect:(id)unused
+{
+  [session connectToTag:tag
+      completionHandler:^(NSError *_Nullable error) {
+        [self fireEvent:@"didConnectTag"
+             withObject:@{
+               @"errorCode" : NUMINTEGER([error code]),
+               @"errorDescription" : [error localizedDescription],
+               @"errorDomain" : [error domain],
+               @"tag" : [[TiNFCTagProxy alloc] _initWithPageContext:[self pageContext] andTag:tag]
+             }];
+      }];
+}
+
+- (NSNumber *)isConnected:(id)unused
+{
+  if (session.connectedTag == tag) {
+    return [NSNumber numberWithBool:YES];
+  } else {
+    return [NSNumber numberWithBool:NO];
```

Comment:
I think the whole condition can be simplified like this 

`return [NSNumber numberWithBool:session.connectedTag == tag?YES:NO];`

Reply:
Updated code.